src/snarky.d.ts

```diff
@@ -51,489 +53,767 @@ declare interface ProvablePure<T> extends Provable<T> {
 }
 
 /**
- * An element of a finite field.
+ * An element of a finite pasta field of order 28948022309329048855892746252171976963363056481941560715954676764349967630337.
  */
-declare function Field(x: Field | number | string | boolean | bigint): Field;
+declare function Field(value: Field | number | string | boolean | bigint): Field;
 declare class Field {
   /**
-   * Coerces anything field-like to a {@link Field}.
+   * Coerce anything `field-like` (bigint, boolean, number, string, and {@link Field}) to a {@link Field}.
+   * A {@link Field} is an element of a prime order field. Every other provable type is build using the {@link Field} type.
+   * The field is the [pasta fp field](https://electriccoin.co/blog/the-pasta-curves-for-halo-2-and-beyond/) of order 28948022309329048855892746252171976963363056481941560715954676764349967630337.
+   * **Warning: You cannot create a {@link Field} from a non-integer number.
+   * 
+   * ```ts
```

Comment:
For all examples, do you mind using the `@example` doclet?
https://jsdoc.app/tags-example.html

Reply:
TIL `@example` doclet!